src/update-changelog.ts

```diff
@@ -213,10 +213,12 @@ async function runCommand(command: string, args: string[]): Promise<string[]> {
  * @param description - The commit message description.
  * @returns The category of the change.
  */
-function getCategory(description: string): ChangeCategory {
-  // Check if description contains a colon
-  if (description.includes(':')) {
-    const [prefix] = description.split(':').map((part) => part.trim());
+export function getCategory(description: string): ChangeCategory {
+  const conventionalCommitPattern = /^(feat|fix)(?:\([^)]*\))?\s*:\s*/u;
```

Comment:
Is there a chance that the section following `^(feat|fix)` gets parsed as a non-capturing group i.e. `(?:pattern)?`?

Reply:
To prevent this, we could escape the enclosing parentheses:

```suggestion
  const conventionalCommitPattern = /^(feat|fix)\(?:\([^)]*\)\)?\s*:\s*/u;
```

Reply:
Ah I can see that I misinterpreted your intention here. Please disregard :)